sycl/include/CL/sycl/feature_test.hpp

```diff
@@ -0,0 +1,18 @@
+//==---- feature_test.hpp - SYCL Feature Test Definitions -----*- C++ -*---==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// ===--------------------------------------------------------------------=== //
+#pragma once
+
+__SYCL_INLINE_NAMESPACE(cl) {
+namespace sycl {
+
+// Feature test macro definitions
```

Comment:
Shouldn't we have an extension doc covering new queries?
@jbrodman @mkinsner

Reply:
llvm/sycl/doc/extensions/IntelGPU/IntelGPUDeviceInfo.md